### PR TITLE
added welcome message to go to new email list subscribers

### PR DESCRIPTION
**Pull request by tracykteal**

### CODE REVIEW

**Comment by maneesha** [comment]
Do we want to make it clear that this is actually a shared space between Data Carpentry and Software Carpentry?

**Comment by tracykteal** [comment]
Thanks good point, updated the message.

**Comment by ctb** [comment]
Looks good to me. The discussion in #111 suggested that we do a few more things somewhere --

* elaborate on who is responsible for dealing with the initial response to a CoC request;
* define a handling and appeals process of some sort;
* provide some more specific (but non-inclusive) positive and negative examples of harassment;

I think this document can be expanded to include those once they are decided upon, so I just wanted to mention them.

**Comment by swaldman3** [comment]
While this is certainly a competent message, it's not quite what I had in mind when I made the suggestion in https://github.com/swcarpentry/board/issues/111.

My suggestion was that a "Welcome to SWC" document should be written that would live on the website, and be linked to new list subscribers. It should be far broader than just covering expected behavior, also explaining how to get involved, what the -discuss list is for, etc. This was aimed at addressing comments that have been raised recently that things are opaque.

Morever, I do not think that the CoC should be contained within this document at all - because one of the reasons for its suggestion was that some people found the formality of the CoC offputting. Instead, IMHO, it should describe the expected behavior briefly, in briefer and less formal terms (even just by the analogy with a professional conference), and link to the CoC.

@ctb I recommend that formalities on how CoC requests, appeals, etc., are handled are definitely not in this document - sending that to everybody who joined the mailing list would certainly not help with the criticism that things are formal and intimidating! If this needs to be codified, it should either go on the same page as the CoC itself or in a seperate document.

**Comment by ctb** [comment]
> @ctb I recommend that formalities on how CoC requests, appeals, etc., are handled are definitely not in this document - sending that to everybody who joined the mailing list would certainly not help with the criticism that things are formal and intimidating! If this needs to be codified, it should either go on the same page as the CoC itself or in a seperate document.

Yep.

**Comment by tracykteal** [comment]
Thanks @swaldman3, a general welcome document on the website is an even better idea. We'll work on that. I started a new issue to figure out what we would want included in such a document. https://github.com/swcarpentry/website/issues/379

I take your point about the CoC. I took it out and we can just link to it. I do think it's important to reference it in the message though.

@ctb thanks, we're working on those processes and will update things on the CoC page when they're in place.

File: welcome.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Welcome to the Software Carpentry email list
+============================================
+
+This is a shared list for Software and Data Carpentry community members.
+
+Software and Data Carpentry are community organizations. We value the involvement of every one in this community - learners, instructors, hosts, developers, steering committee members and staff. We are committed to creating a friendly and respectful place for learning, teaching and contributing. All participants in Software Carpentry events or communications are expected to show respect and courtesy to others.
+
+To make clear what is expected, everyone participating in Software and Data Carpentry activities is required to conform to the following Code of Conduct. This code of conduct applies to all spaces managed by Software Carpentry, including, but not limited to, workshops, email lists, online forums and on GitHub.
```
Comment (wking) [inline welcome.md]: “the following Code of Conduct” should be “our [Code of Conduct](https://software-carpentry.org/conduct)”.  And if we go with caps there, I suggest the following sentence start with “This Code of Conduct applies…”.

**Comment by karinlag** [comment]
So.... let's talk about a toc for this kind of thing:

First, I think that what @tracykteal would be a great intro to this page. We then need:

What we are - link to mission statement, history etc
What we do - link to workshop stuff and instructor training
Who we are - link to board, instructor member list, partnership list etc
How to interact - website, email lists, facebook, twitter etc
Rules of interaction - link to CoC etc

Of course, each of these need to come with a sentence or two to make sense of it all, but ideally this should not be more than one (probably slightly lengthy) paragraph, which could then go both on the website and be blasted out whenever somebody signs up for an email list.

Anything else that should be included...?

**Comment by gvwilson** [comment]
This one has been sitting for a while - anyone want to push it over the finish line?

**Comment by swaldman3** [comment]
I think given the ongoing work on the CoC document, it might be worth waiting until that is done. 
(and yeah, I don't have time right now and this is an excuse ;-))

**Comment by karinlag** [comment]
On 15.08.2016 04:42, Simon Waldman wrote:
> I think given the ongoing work on the CoC document, it might be worth
> waiting until that is done.
> (and yeah, I don't have time right now and this is an excuse ;-))

I agree - personally I was thinking of giving this a go, but it does 
seem sensible to pause this one until the CoC committee is up and running.

But... we could make a stab at something non-CoC related, such as a bit 
about the carpentries and some useful links. I've also been thinking 
about having something(TM) along those lines in the footer of the 
Discuss email list emails - not more than a sentence or two, but just 
some pointers.

Karin

**Comment by jduckles** [comment]
Any objection to closing this one as outdated?

**Comment by wking** [comment]
On Tue, Dec 20, 2016 at 11:06:37AM -0800, Jonah Duckles wrote:
> Any objection to closing this one as outdated?

This PR still looks reasonably fresh to me.  I think having a
carpentry-specific welcome message for discuss@ (and our other lists?)
still makes sense.

**Comment by jduckles** [comment]
@wking, ok then it needs to 1) be brought up to date with the current policy as @swaldman3 said and 2) have its language modified to be pan-carpentries.

**Comment by wking** [comment]
On Wed, Dec 21, 2016 at 06:59:40AM -0800, Jonah Duckles wrote:
> ok then it needs to 1) be brought up to date with the current policy
> as @swaldman3 said and 2) have its language modified to be
> pan-carpentries.

@tracykteal, do you want to pick those^ up?  Or would you rather I
submitted a separate PR with an additional commit on top of yours?

**Comment by karinlag** [comment]
On 21.12.2016 21:12, W. Trevor King wrote:
> On Wed, Dec 21, 2016 at 06:59:40AM -0800, Jonah Duckles wrote:
>> ok then it needs to 1) be brought up to date with the current policy
>> as @swaldman3 said and 2) have its language modified to be
>> pan-carpentries.
>
> @tracykteal, do you want to pick those^ up? Or would you rather I
> submitted a separate PR with an additional commit on top of yours?


I can give this a go a bit later in the week.

Karin

**Pull request merged**